### PR TITLE
feat: ami describe by name

### PR DESCRIPTION
**Pull request by tekumara**
eg:
```
❯ ami describe "AWS Deep Learning Base AMI GPU CUDA 11 (Ubuntu 20.04) 20230314"
Describing images owned by ['self', '099720109477', '898082745236'] named AWS Deep Learning Base AMI GPU CUDA 11 (Ubuntu 20.04) 20230314
                                                                                                                                             
  Name                                                             ImageId                 CreationDate               RootDeviceName   Size  
 ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── 
  AWS Deep Learning Base AMI GPU CUDA 11 (Ubuntu 20.04) 20230314   ami-0a50ae3a720737240   2023-03-15T11:10:41.000Z   /dev/sda1        100   
```

### CODE REVIEW

File: src/aec/command/ami.py [hunk 1]
```diff
@@ -51,56 +51,59 @@ def fetch(config: Config, ami: str) -> Image:
...
 
-        response = ec2_client.describe_images(Owners=owners_filter, Filters=filters)
-    return response
+    return ec2_client.describe_images(Owners=owners_filter, Filters=filters)
 
 
 def describe(
     config: Config,
-    id: Optional[str] = None,
+    name: Optional[str] = None,
```
Comment (mhasanbulli) [inline src/aec/command/ami.py:99]: Nit: We still can provide an id here. The name of the variable is not so obvious here. Hmmm... 🤔

**Review by mhasanbulli — APPROVED** [review]
LGTM 🚀 🎸

File: src/aec/command/ami.py (hunk 1 above)
Comment (tekumara) [reply src/aec/command/ami.py:99]: yeh, I did consider calling it `name_or_id` ... maybe that's better?

Comment (mhasanbulli) [reply src/aec/command/ami.py:99]: `name_or_id` is fine for the internal code. Though it would look pretty ugly for the CLI, wouldn't it? 🤣

Comment (tekumara) [reply src/aec/command/ami.py:99]: yeh it's a bit ugly which is why i was a little hesitant...

Comment (tekumara) [reply src/aec/command/ami.py:99]: could call it `ident`?

Comment (mhasanbulli) [reply src/aec/command/ami.py:99]: Short for identity?

Comment (tekumara) [reply src/aec/command/ami.py:99]: identifier

Comment (tekumara) [reply src/aec/command/ami.py:99]: which could be a name or an id

Comment (mhasanbulli) [reply src/aec/command/ami.py:99]: Yeah. I guess it is fine, until you come up with something new 🤣 Naming things are really hard!!

Comment (tekumara) [reply src/aec/command/ami.py:99]: 👍 i'll update that in a separate refactor PR

**Pull request merged**